gempython/tools/amc_user_functions_xhal.py

```diff
@@ -179,6 +179,12 @@ def __init__(self, cardName, debug=False):
         if debug:
             print "My FW release major = ", self.fwVersion
 
+        #Determine the number of GBTs based on the gemType
```

Comment:
I don't see how the gemType is passed to the constructor of the `HwAMC` class. Please update the `init` signature

Reply:
Fixed! Thank you for spotting that.